core/pom.xml

```diff
@@ -194,12 +194,6 @@
 			<version>1.10.0</version>
 		</dependency>
 
-<!-- 	(old) digester is used in the StatisticsParser -->
-		<dependency>
-			<groupId>commons-digester</groupId>
-			<artifactId>commons-digester</artifactId>
-			<version>2.1</version>
-		</dependency>
 		<!-- Requirement for Jboss 7.5 -->
```

Comment:
"Requirement for JBoss 7.5" -- knap, want we zitten pas net op 7.4! 🤣

Reply:
Ja ik zag het ook, geen idee :S